web/client/src/library/components/plan/PlanActions.tsx

```diff
@@ -105,6 +109,8 @@ export default function PlanActions({
     isFalse(planAction.isCancelling) &&
     isFalse(planAction.isDone) &&
     (planAction.isProcessing ? isFalse(planCancel.isSuccessed) : true)
+  const isFailedOrCanceled =
+    planOverview.isFailed || planApply.isFailed || planCancel.isSuccessed
```

Comment:
Nit: I just noticed this. Sorry I didn't catch it before. isSuccessful? hasSucceeded?